db/data/regions/_default/en.yml

```diff
@@ -20,69 +21,77 @@ en:
               local characters, and special characters
-            contains_too_many_words: Address1 cannot have more than %{word_count}
+            contains_too_many_words: Address1 must have fewer than %{word_count} words
               words
-            contains_html_tags: Address1 cannot contain HTML tags.
-            contains_url: Address1 cannot contain URLs.
-            missing_building_number: Add a building number if you have one.
-            street_unknown_for_zip: Enter a valid street name for %{zip}.
-            building_number_invalid: Building number couldn't be located for %{street},
-              %{zip}.
-            unknown_for_address: Address line 1 may be incorrect.
+            contains_html_tags: Address1 can't contain HTML
+            contains_url: Address1 can't contain a URL
+            missing_building_number_instructional: Add a building number if you have one
+            missing_building_number_informative: Building number may be missing
+            street_unknown_for_zip_instructional: Enter a valid street name for %{zip}.
+            street_unknown_for_zip_informative: Street name doesn't match %{zip}
+            building_number_invalid_instructional: Building number couldn't be located for %{street}, %{zip}
+            building_number_invalid_informative: Building number couldn't be located for %{street}, %{zip}
+            unknown_for_address: Address line 1 may be incorrect
           warnings:
             contains_too_many_words: Address line 1 is recommended to have less than %{word_count} words
         address2:
           label:
             default: Apartment, suite, etc.
             optional: Apartment, suite, etc. (optional)
           errors:
-            blank: Enter an apartment, suite, etc.
+            blank_instructional: Enter an apartment, suite, etc.
+            blank_instructional: Apartment, suite, etc is missing
             too_long: The second address line is too long (maximum is 255 characters)
             contains_emojis: Address2 cannot contain emojis
             contains_mathematical_symbols: Address2 cannot contain mathematical symbols
             contains_restricted_characters: Address2 can only contain letters, numbers,
               local characters, and special characters
             contains_too_many_words: Address2 cannot have more than %{word_count}
               words
-            contains_html_tags: Address2 cannot contain HTML tags.
-            contains_url: Address2 cannot contain URLs.
-            missing_building_number: Add a building number if you have one.
-            street_unknown_for_zip: Enter a valid street name for %{zip}.
-            building_number_invalid: Building number couldn't be located for %{street},
-              %{zip}.
-            unknown_for_address: Address line 2 may be incorrect.
+            contains_html_tags: Address2 can't contain HTML
+            contains_url: Address2 can't contain a URL
```

Comment:
Again, what is the field label? If we have to use this term, then we should say `Address line 2`, not `Address2`